### PR TITLE
Ian UID2-1123 cstg keypair UI

### PR DESCRIPTION
**Pull request by Ian-Nara**
[UID2-1123](https://atlassian.thetradedesk.com/jira/browse/UID2-1123) 
UI for CRUD Keypair manipulation

### CODE REVIEW

**Review by cody-constine-ttd — APPROVED** [review]
LGTM

File: webroot/index.html [hunk 1]
```diff
@@ -24,6 +24,7 @@ <h1>UID2 Admin Site</h1>
     <li class="ro-cki ro-adm" style="display: none"><a href="/adm/key-acl.html">Encryption Key ACL Management</a></li>
     <li class="ro-adm" style="display: none"><a href="/adm/keysets.html">Keyset Access Management</a></li>
     <li class="ro-sem" style="display: none"><a href="/adm/encryption-key.html">Encryption Key Management</a></li>
+    <li class="ro-adm" style="display: none"><a href="/adm/client-side-keypairs.html">Client Side Keypair Management</a></li>
```
[the remark this answers is not in the document]
Comment (sunnywu) [reply webroot/index.html:27]: sorry i left this comment a couple of days ago but didn't publish it

**Comment by sunnywu** [comment]
LGTM apart from my remaining comment.

File: webroot/js/main.js [hunk 2]
```diff
@@ -39,12 +37,16 @@ function doApiCallWithBody(method, url, body, outputDiv, errorDiv) {
...
-            $(errorDiv).html('Error: ' + err.status + ': ' + (isJsonString(err.responseText) ? JSON.parse(err.responseText).message : (err.responseText ? err.responseText : err.statusText)));
-        }
+        error: function (err) { standardErrorCallback(err, outputDiv) }
     });
 }
 
+function errorCallback(err) { standardErrorCallback(err, '#errorOutput') }
+
+function standardErrorCallback(err, errorDiv) {
+    $(errorDiv).html('Error: ' + err.status + ': ' + (isJsonString(err.responseText) ? JSON.parse(err.responseText).message : (err.responseText ? err.responseText : err.statusText)));
```
Comment (mcollins-ttd) [inline webroot/js/main.js:47]: Here and elsewhere, prefer `text()` to `html()` if we're just setting text.

**Review by mcollins-ttd — APPROVED** [review]

File: webroot/js/main.js (hunk 2 above)
Comment (Ian-Nara) [reply webroot/js/main.js]: switched to text()

File: webroot/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ian-Nara) [reply webroot/index.html]: sounds good! updated

**Review by sunnywu — APPROVED** [review]

**Pull request merged**